verifier-guest.sh

```diff
@@ -113,11 +113,16 @@ sudo service postgresql restart
 
 #install numpy
 pip install numpy
-pip install shapely==1.5.13
-
 
 ## Clone GeoNode
-git clone --depth=1 -b "$GIT_GEO_REPO" https://github.com/GeoNode/geonode.git
+if [ "$GEM_TEST_LATEST" = "true" ]; then
+    git clone --depth=1 -b "$GIT_GEO_REPO" https://github.com/GeoNode/geonode.git
+else
+    git clone -n https://github.com/GeoNode/geonode.git
+    cd geonode
+    git checkout 1c65c9b
```

Comment:
Maybe add a variable at top of the script that will be used here could be a good option.

Reply:
Changed here: https://github.com/gem/oq-platform2/pull/15/commits/2b0e86ed7c62fcbf7b047b1c91bc1a3d15bfa5b0